packages/core/src/prisma/parse.ts

```diff
@@ -0,0 +1,26 @@
+import { readFileSync } from "fs";
+import { parsePrismaSchema as schemaParser } from "@loancrate/prisma-schema-parser";
+
+export type getModelFromSchemaOptions = {
+  schemaPath?: string;
+};
+
+const getModelFromSchemaDefaults: getModelFromSchemaOptions = {
+  schemaPath: "prisma/schema.prisma",
+};
+
+export const getModelFromSchema = (
+  modelName: string,
+  options?: getModelFromSchemaOptions
+) => {
+  const resolvedOptions = {
+    ...getModelFromSchemaDefaults,
+    ...options,
+  };
+  const file = readFileSync(resolvedOptions.schemaPath, { encoding: "utf-8" });
```

Comment:
How would we handle setting the defaults in a better way? The current implementation throws an typescript error in line `20`